src/procedures/__tests__/LaunchTieredSto.ts

```diff
@@ -44,6 +44,8 @@ const params: LaunchTieredStoProcedureArgs = {
   minimumInvestment: new BigNumber(1),
   currencies: [Currency.StableCoin],
   stableCoinAddresses: ['0x7777777777777777777777777777777777777777'],
+  customOracleAddresses: ['0x8888888888888888888888888888888888888888'],
+  denominatedCurrency: 'USDT',
```

Comment:
denominatedCurrency doesn't refer to the stablecoins, but to the actual currency, so here it would be 'USD'